### ISSUE TITLE
Brew install warning: No --with-pinentry option for formula

### ISSUE THREAD

**Issue by alichtman** [lastpass/lastpass-cli#470]
This warning caught my eye, since this is the command recommended to be run in the README.

`Warning: lastpass-cli: this formula has no --with-pinentry option so it will be ignored!`

Should there be a formla with this option? Or should the installation section of the README be updated?

```zsh
➜ brew install lastpass-cli --with-pinentry                                                                                                
Updating Homebrew...
==> Auto-updated Homebrew!
Updated Homebrew from 9141b1509 to da2219419.
Updated 3 taps (homebrew/core, homebrew/cask and homebrew/bundle).
==> Updated Formulae
...

==> Installing dependencies for lastpass-cli: curl
==> Installing lastpass-cli dependency: curl
==> Downloading https://homebrew.bintray.com/bottles/curl-7.63.0.mojave.bottle.1.tar.gz
######################################################################## 100.0%
==> Pouring curl-7.63.0.mojave.bottle.1.tar.gz
==> Caveats
curl is keg-only, which means it was not symlinked into /usr/local,
because macOS already provides this software and installing another version in
parallel can cause all kinds of trouble.

If you need to have curl first in your PATH run:
  echo 'export PATH="/usr/local/opt/curl/bin:$PATH"' >> ~/.zshrc

For compilers to find curl you may need to set:
  export LDFLAGS="-L/usr/local/opt/curl/lib"
  export CPPFLAGS="-I/usr/local/opt/curl/include"

For pkg-config to find curl you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/curl/lib/pkgconfig"


zsh completions have been installed to:
  /usr/local/opt/curl/share/zsh/site-functions
==> Summary
🍺  /usr/local/Cellar/curl/7.63.0: 446 files, 3.2MB
Warning: lastpass-cli: this formula has no --with-pinentry option so it will be ignored!
==> Installing lastpass-cli
==> Downloading https://homebrew.bintray.com/bottles/lastpass-cli-1.3.1_1.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring lastpass-cli-1.3.1_1.mojave.bottle.tar.gz
==> Caveats
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completions have been installed to:
  /usr/local/share/zsh/site-functions
==> Summary
🍺  /usr/local/Cellar/lastpass-cli/1.3.1_1: 10 files, 215.1KB
==> Caveats
==> curl
curl is keg-only, which means it was not symlinked into /usr/local,
because macOS already provides this software and installing another version in
parallel can cause all kinds of trouble.

If you need to have curl first in your PATH run:
  echo 'export PATH="/usr/local/opt/curl/bin:$PATH"' >> ~/.zshrc

For compilers to find curl you may need to set:
  export LDFLAGS="-L/usr/local/opt/curl/lib"
  export CPPFLAGS="-I/usr/local/opt/curl/include"

For pkg-config to find curl you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/curl/lib/pkgconfig"


zsh completions have been installed to:
  /usr/local/opt/curl/share/zsh/site-functions
==> lastpass-cli
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completions have been installed to:
  /usr/local/share/zsh/site-functions
``

**Comment by thausler786** [comment]
I also encountered this problem.

**Comment by parsingphase** [comment]
Issue is present in macOS Mojave 10.14.3 (18D109),
Homebrew 2.0.3
Homebrew/homebrew-core (git revision beb6; last commit 2019-03-06)

**Comment by tklovett** [comment]
It appears that `pinentry` is now installed by default. It gets installed even when you omit the `--with-pinentry` flag:

```bash
$ brew -v
Homebrew 2.0.3-41-g0f07fe5
Homebrew/homebrew-core (git revision 6a886; last commit 2019-03-12)
Homebrew/homebrew-cask (git revision d746f; last commit 2019-03-11)

$ brew install lastpass-cli
==> Installing dependencies for lastpass-cli: curl, libgpg-error, libassuan and pinentry
...
```

**Comment by alichtman** [comment]
Good find, @tklovett.

**Comment by alichtman** [comment]
Closed in #480
